athena-federation-sdk/pom.xml

```diff
@@ -22,7 +22,6 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <slf4jVersion>1.7.30</slf4jVersion>
         <log4j2Version>2.13.3</log4j2Version>
-        <aws-sdk.version>1.11.769</aws-sdk.version>
```

Comment:
why was this removed?

Reply:
The variable was duplicated from the main app pom causing a conflict situation where the Federation SDK could have a different AWS SDK version than the Connectors.